sample-material/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.application'
```

Comment:
I don't think we need a sample app specifically for material, you can use the existing sample app.